raiden/network/proxies/token_network.py

```diff
@@ -242,30 +246,34 @@ def detail_channel(self, partner: typing.Address) -> Dict:
             'state': channel_data[2],
         }
 
-    def detail_participants(self, partner: typing.Address) -> Dict:
+    def detail_participants(
+            self,
+            participant1: typing.Address,
+            participant2: typing.Address,
+    ) -> Dict:
         """ Returns a dictionary with the participants' channel information. """
-        our_data = self.detail_participant(self.node_address, partner)
-        partner_data = self.detail_participant(partner, self.node_address)
+        our_data = self.detail_participant(participant1, participant2)
+        partner_data = self.detail_participant(participant2, participant1)
```

Comment:
these values could be swapped around, right? what about adding a check against `self.node_address`?